__tests__/__snapshots__/App-test.js.snap

```diff
@@ -79,7 +79,7 @@ exports[`App renders the loading screen 1`] = `
               }
               style={
                 Object {
-                  "flex": 0.2,
+                  "height": 70,
```

Comment:
The only thing that worries me about using a fixed height is how it will look on other/smaller devices, but it's great for now!